apps/docs/project.json

```diff
@@ -197,50 +197,23 @@
                     "sourceMap": false,
                     "namedChunks": false,
                     "vendorChunk": false
-                },
-                "hmr": {
-                    "budgets": [
-                        {
-                            "type": "anyComponentStyle",
-                            "maximumWarning": "6kb"
-                        }
-                    ],
-                    "optimization": true,
-                    "outputHashing": "all",
-                    "sourceMap": false,
-                    "namedChunks": false,
-                    "extractLicenses": true,
-                    "vendorChunk": false,
-                    "buildOptimizer": true,
-                    "fileReplacements": [
-                        {
-                            "replace": "apps/docs/src/environments/environment.ts",
-                            "with": "apps/docs/src/environments/environment.prod.ts"
-                        }
-                    ]
                 }
             },
             "outputs": ["{options.outputPath}"],
             "defaultConfiguration": "",
             "dependsOn": ["^build", "generate-typedoc"]
         },
         "serve": {
-            "executor": "@angular-devkit/build-angular:dev-server",
```

Comment:
`@angular-devkit/build-angular:dev-server` can be removed from package.json,
as well as `@angularclass/hmr`. also main.ts should be updated.

Reply:
Lets merge this first and then I'll update https://github.com/SAP/fundamental-ngx/pull/8698